src/test/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModificationTest.java

```diff
@@ -74,6 +74,38 @@ public void testWrongVoltageLevelId() {
                 shuntCompensator.getErrorType().name(), reportService);
     }
 
+    @SneakyThrows
+    @Test
+    public void testWrongMaximumSectionCount() {
+        var shuntCompensator = ShuntCompensatorModificationInfos.builder()
+                .equipmentId("v5shunt")
+                .sectionCount(new AttributeModification<>(3, OperationType.SET))
+                .maximumSectionCount(new AttributeModification<>(-1, OperationType.SET))
+                .build();
+
+        mockMvc.perform(post(getNetworkModificationUri()).content(mapper.writeValueAsString(shuntCompensator)).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+        assertLogMessage(new NetworkModificationException(MODIFY_SHUNT_COMPENSATOR_ERROR,
+                        String.format("Maximum section count should be greater or equal to 1")).getMessage(),
+                shuntCompensator.getErrorType().name(), reportService);
+    }
+
+    @SneakyThrows
+    @Test
+    public void testWrongSectionCount() {
```

Comment:
Maybe it's missing a test where you only modify `sectionCount` ? Your code is supposed to work but it's not tested